src/main/resources/application-dev.yml

```diff
@@ -9,14 +9,16 @@ api:
       #oauth: http://localhost:9090/auth
       prison: http://localhost:8083
       #prison: https://prison-register-dev.hmpps.service.justice.gov.uk
-      #offender-search: https://prisoner-offender-search-dev.prison.service.justice.gov.uk
-      offender-search: http://localhost:8085
+      offender-search: https://prisoner-offender-search-dev.prison.service.justice.gov.uk
+      #offender-search: http://localhost:8085
       cvl: http://localhost:8081
       #cvl: https://create-and-vary-a-licence-api-dev.hmpps.service.justice.gov.uk
       community: http://localhost:8082
       #community: https://community-api.test.probation.service.justice.gov.uk
       arn: http://localhost:8084
       #arn: https://assess-risks-and-needs-dev.hmpps.service.justice.gov.uk
+      #offender-image: http://localhost:8086
```

Comment:
As per the naming convention this should be the name of the service we are calling not the purpose